packages/module/src/ConsoleInternal/vendor.scss

```diff
@@ -1,6 +1,5 @@
-@import 'style/vars';
-@import '~patternfly/dist/sass/patternfly/variables';
-@import '~bootstrap-sass/assets/stylesheets/bootstrap/variables';
-@import '~bootstrap-sass/assets/stylesheets/bootstrap/mixins';
-@import '~patternfly/dist/sass/patternfly/bootstrap-mixin-overrides';
+@import '~@patternfly/patternfly/sass-utilities/all';
+
 @import '~@patternfly/react-catalog-view-extension/dist/sass/react-catalog-view-extension.scss';
```

Comment:
These styles are no longer available now that [the dependency has been cut](https://github.com/patternfly/patternfly-quickstarts/pull/25/files#diff-75883ecc6980c44f5eae3e923c7ce65e04db292389d9f2a54dd1a759c8217b40L48), no?

Reply:
Thanks, it was still being pulled into node_modules implicitly, but I'll add it back in as a dep